site/watch.js

```diff
@@ -41,14 +49,14 @@ function getFullPath(filePath) {
 
 function syncFile(filePath) {
   const { src, destination } = getFullPath(filePath)
-  return fs.copy(src, destination).then(() => {
+  return fs.copyFile(src, destination).then(() => {
```

Comment:
⚠ This is not recursive like the one from `fs-extra`

I don't think this is really an issue here as it will only be called with file paths.